playbook/app/pb_kits/playbook/pb_dropdown/dropdown.rb

```diff
@@ -10,6 +10,7 @@ class Dropdown < Playbook::KitBase
       prop :error, type: Playbook::Props::String
       prop :required, type: Playbook::Props::Boolean,
                       default: false
+      prop :defaultValue
```

Comment:
Everything looks great @carloslimasd ! Minor feedback: as per rails kits pattern, this prop name should be default_value on rails side

Reply:
Thank you for the feedback!